qctrlopencontrols/dynamic_decoupling_sequences/driven_controls.py

```diff
@@ -206,6 +206,16 @@ def convert_dds_to_driven_control(
         azimuthal_angles = np.append(azimuthal_angles, [0])
         detuning_rotations = np.append(detuning_rotations, [0])
 
+    # check that the offsets are correctly sorted in time
+    time_differences = np.diff(offsets)
+    if not np.all(np.logical_or(np.greater(time_differences, 0.),
+                                np.isclose(time_differences, 0.))):
+        raise ArgumentsValueError("Pulse timing could not be properly deduced from "
+                                  "the sequence offsets. Make sure all offset are "
+                                  "correctly ordered in time.",
```

Comment:
Instead of saying "correctly ordered in time", how about we just say "increasing order"? That would be a more clear indication of how the user can fix the issue.